portal/code/cx-portal-shared-components/src/components/basic/Checkbox/index.tsx

```diff
@@ -1,13 +1,17 @@
 import MuiCheckbox, {
   CheckboxProps as MuiCheckboxProps,
 } from '@mui/material/Checkbox'
+import FormControlLabel from '@mui/material/FormControlLabel';
 
-const label = { inputProps: { 'aria-label': 'Checkbox demo' } }
+const ariaLabel = { inputProps: { 'aria-label': 'Checkbox demo' } }
 
 interface CheckboxProps extends Omit<MuiCheckboxProps, 'size'> {
-  size?: 'medium' | 'small'
+  size?: 'medium' | 'small',
+  label?: string | number
 }
 
-export const Checkbox = ({ size = 'medium', ...props }: CheckboxProps) => {
-  return <MuiCheckbox size={size} {...props} {...label} />
+export const Checkbox = ({ size = 'medium', label, ...props }: CheckboxProps) => {
+  return label 
+  ? <FormControlLabel disabled control={<MuiCheckbox size={size} {...props} {...ariaLabel} />} label={label} /> 
```

Comment:
in Storybook the checkbox seems to be disabled permanently - maybe from this setting?

Reply:
Yeeees! Thank you Martin :) 
It's removed now

Reply:
thanks - working now ;)